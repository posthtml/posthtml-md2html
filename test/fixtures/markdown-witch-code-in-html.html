<div>
  <md>
    ```html
    Unmarked title
    <div>Content</div>
    More content
    ```
  </md>
</div>