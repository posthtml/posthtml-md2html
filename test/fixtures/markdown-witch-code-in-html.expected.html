<div>
  <pre><code class="language-html">    Unmarked title
    &lt;div&gt;Content&lt;/div&gt;
    More content</code></pre>

</div>